semgrep-core/src/mlgrep/mlgrep-util/Constants.ml

```diff
@@ -0,0 +1,152 @@
+(*
```

Comment:
not a big fan of -util suffix for directories.

Reply:
and not a big fan of mlgrep- prefix also. maybe just semgrep-core/src/semgrep-ocaml/core/
that contains the common stuff.